lib/hanami/router/route.rb

```diff
@@ -5,24 +5,27 @@
 
 module Hanami
   class Router
+    # A route from the router
+    #
+    # @since 2.0.0
     class Route
       # @api private
+      # @since 2.0.0
+      ROUTE_CONSTRAINT_SEPARATOR = ", "
```

Comment:
@waiting-for-dev Leftover?

Reply:
Actually, it's still [used in the same file](https://github.com/hanami/router/blob/4caec5d79f2449d8bf274d2bac319930522423d7/lib/hanami/router/route.rb#L76) to inspect the constraints.

Reply:
In fact, it was a leftover in `Hanami::Router::Formatter::HumanFriendly`. I just removed it from there.